server/api/pipeline.go

```diff
@@ -638,7 +641,7 @@ func PostPipeline(c *gin.Context) {
 			return
 		}
 
-		pl.DeployTo = c.DefaultQuery("deploy_to", pl.DeployTo)
+		pl.Deployment.Target = c.DefaultQuery("deploy_to", pl.Deployment.Target)
```

Comment:
This property is called `deployment.target` now while the query parameter is `deploy_to`. Should use one term to be consistent with a deprecation and fallback if necessary.